doc/src/modules/algebras.rst

```diff
@@ -15,5 +15,9 @@ Quaternion Reference
 
 This section lists the classes implemented by the Algebras module.
 
+.. class:: SExpr
+.. class:: Matrix
+.. class:: SymbolLimits
+
 .. autoclass:: Quaternion
```

Comment:
There might be a better way than listing these type aliases as classes here. I think it is a sphinx bug that an annotation using a type alias doesn't resolve: https://github.com/sphinx-doc/sphinx/issues/10785